beep/structure/tests/test_cyclerpaths.py

```diff
@@ -358,9 +358,42 @@ def test_from_csv(self):
         self.assertAlmostEqual(dp.structured_data["test_time"].max(), 101972.885, 3)
 
     def test_from_txt(self):
```

Comment:
Let's keep the existing test and just create a new test for this bug. If some of the tests are failing we can take a look at what needs to be altered to make them pass.

Reply:
Resolved. Updated the existing tests and created a new test.